gen/flights/flightplan.py

```diff
@@ -114,6 +116,51 @@ def tot_waypoint(self) -> Optional[FlightWaypoint]:
         failed to generate. Nevertheless, we have to defend against it.
         """
         raise NotImplementedError
+    
+    @cached_property
+    def bingo_fuel(self) -> int:
+        """Bingo fuel value for the FlightPlan
+        """
+        assert self.farthest_wpt_from_arrival is not None
+        distanceToArrival = meter_to_nm(self.farthest_wpt_from_arrival)
+
+        bingo = 1000 # Minimum Emergency Fuel
+        bingo += 500 # Visual Traffic
+        bingo += 15 * distanceToArrival
+
+        if self.farthest_wpt_from_divert is not None:
```

Comment:
And this can skip the asserts with:

```
if self.flight.divert is not None:
    bingo += 10 * meter_to_nm(self.max_distance_from(self.flight.divert))
```

That way we don't need to worry about `None` being being returned from